web/src/pages/guidance/topic-selection.page.tsx

```diff
@@ -0,0 +1,64 @@
+import { SortOrder, Document } from "@nice-digital/search-client";
+
+import {
+	getGuidanceListPage,
+	getGetServerSidePropsFunc,
+} from "@/components/GuidanceListPage/GuidanceListPage";
+
+const defaultSortOrder = SortOrder.titleAscending;
+
+const tableBodyRender = (documents: Document[]) => (
+	<>
+		<caption className="visually-hidden">Guidance in topic selection</caption>
+		<thead>
+			<tr>
+				<th scope="col">Title</th>
+				<th scope="col">Technology type</th>
+				<th scope="col">Decision</th>
+			</tr>
+		</thead>
+		<tbody>
+			{documents.map(
+				({
+					id,
+					title,
+					pathAndQuery,
+					technologyType,
+					topicSelectionDecision,
+				}) => {
+					return (
+						<tr key={id}>
+							<td>
+								<a
+									href={pathAndQuery}
```

Comment:
Is there a potential here for an XSS attack since the `pathAndQuery` is not in a quoted attribute? So it maybe possible to escape out of the attribute.

Reply:
the `pathAndQuery` variable comes from SSE so isn't user content AND it's JSX which will escape things: https://reactjs.org/docs/introducing-jsx.html#jsx-prevents-injection-attacks